qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -243,6 +243,15 @@
             only all_drivers
             driver_test_names = ''
             type = win_virtio_driver_installer_uninstall
+            variants:
+                - by_installer:
+                - by_msi:
+                    required_virtio_win = [1.9.35.0, )
+                    uninstall_method = msi
+                    run_uninstall_cmd = 'WIN_UTILS:\AutoIt3_%PROCESSOR_ARCHITECTURE%.exe WIN_UTILS:\uninstall_via_msi.au3'
+                    warning_text_path = "C:\uninstall_warning.txt"
+                    check_warning_file = 'type ${warning_text_path}'
+                    warning_message = ''
```

Comment:
Should we set a default value for it?

Reply:
Hi @vivianQizhu 

Sure, I also prefer to set a expected warning message. 
like that:https://github.com/autotest/tp-qemu/commit/3685f30bfda7b2b8ab85d3522f08bc57b835d121